virtcontainers/api.go

```diff
@@ -758,6 +758,46 @@ func StatsContainer(ctx context.Context, sandboxID, containerID string) (Contain
 	return s.StatsContainer(containerID)
 }
 
+// StatsSandbox is the virtcontainers sandox stats entry point.
```

Comment:
```suggestion
// StatsSandbox is the virtcontainers sandox stats entry point.
```
// StatsSandbox is the virtcontainers sandbox stats entry point.